collectors/nvd/collectors.py

```diff
@@ -284,6 +284,10 @@ def update_cvss_via_flawcvss(self, flaw: Flaw, item: dict) -> bool:
                     flaw.cvss_scores.filter(issuer=FlawCVSS.CVSSIssuer.NIST).filter(
                         version=version
                     ).delete()
+                    # remove also any potential NIST CVSS validation flag
+                    Flaw.objects.filter(uuid=flaw.uuid).update(
```

Comment:
I am not sure I understand the first paragraph - the second will be addressed based on the thread above. Are you arguing that it is changing the timestamp or that it is not?

I am using **.update** instead of **.save** to make it atomic. On the other hand your comment makes me think about whether it is actually enough as it would not update the timestamp unlike **.save** and eg. that SDEngine could not know that a flaw got changed - a score was removed. However, if so, that is actually another and already existing issue. I am only dealing with the flag here and not flaw timestamps.

Reply:
But I agree that the timestamp should get update on flaw update. A general solution would be better - not just adding a new parameter here but something maybe on the tracking mixin level.

Reply:
My point was that your change might update flaw timestamps more times than we would like. But I probably wrongly understand behaviour of `.update`, and it does not change timestamps at all.

Reply:
@jobselko the timestamp issue is more complicated and probably already ongoing plus incorporating the signals too. I am creating a separate Jira for it https://issues.redhat.com/browse/OSIDB-3573, is it OK?

Reply:
Sure, thank you! :)